frontend/src/views/SQR/Form.vue

```diff
@@ -527,7 +527,7 @@ export default class Form extends Vue {
         .catch(() => "error");
 
       if (updated === "error") {
-        this.draftStatus = FormState.ERROR;
+        this.formStatus = FormState.ERROR;
         this.formStateMessages[FormState.ERROR] = "Permission denied";
```

Comment:
There is no other place where you set another message for `FormState.ERROR`. Consider setting this message in the `formStateMessages` definition once above in the line L293.